### PR TITLE
Make it possible to run spectests with engine-native.

### PR DESCRIPTION
**Pull request by nlewycky**

### CODE REVIEW

File: tests/compilers/wast.rs [hunk 1]
```diff
@@ -21,13 +21,6 @@ use wasmer_wast::Wast;
 // }
 include!(concat!(env!("OUT_DIR"), "/generated_spectests.rs"));
 
-// This prefixer returns the hash of the module to prefix each of
-// the functions in the shared object generated by the `NativeEngine`.
-fn _native_prefixer(bytes: &[u8]) -> String {
-    let hash = blake3::hash(bytes);
-    format!("{}", hash.to_hex())
-}
-
```
Comment (syrusakbary) [inline tests/compilers/wast.rs:30]: Can we keep this function, it might be useful for the future

Comment (nlewycky) [reply tests/compilers/wast.rs]: Done.

File: tests/compilers/utils.rs [hunk 2]
```diff
@@ -32,9 +35,17 @@ pub fn get_compiler(canonicalize_nans: bool) -> impl CompilerConfig {
     }
 }
 
+#[cfg(feature = "test-jit")]
 pub fn get_engine() -> impl Engine {
     let compiler_config = get_compiler(false);
     JIT::new(&compiler_config).engine()
+    // let mut compiler_config = get_compiler(false);
+    // Native::new(&mut compiler_config).engine()
```
Comment (syrusakbary) [inline tests/compilers/utils.rs:43]: We should remove this

**Review by syrusakbary — APPROVED** [review]

File: tests/compilers/utils.rs (hunk 2 above)
Comment (nlewycky) [reply tests/compilers/utils.rs]: Done.

**Comment by nlewycky** [comment]
FYI, this introduced a failure in tarpaulin runs because it didn't add the `test-jit` feature to `.tarpaulin.toml` which is now needed to call `get_headless_store` in tests/compiler/utils.rs.

**Pull request merged**